tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/game_preferences/play_fov_effects.tsx

```diff
@@ -0,0 +1,12 @@
+import { multiline } from "common/string";
+import { CheckboxInput, FeatureToggle } from "../base";
+
+export const play_fov_effects: FeatureToggle = {
```

Comment:
why is this a preference?

Reply:
This is the preference for the visual effects inside FOV being played (footsteps, attacks). Should it be not? I can remove it

Reply:
I don't see any reason for it to be a preference no

Reply:
yeah why on earthw ould you ever disable it, no need.

Reply:
I shall remove it then